<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LXQtClockConfiguration</class>
 <widget class="QDialog" name="LXQtClockConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clock Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="timeGB">
     <property name="title">
      <string>Time</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="showSecondsCB">
        <property name="text">
         <string>&amp;Show seconds</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ampmClockCB">
        <property name="text">
         <string>12 &amp;hour style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="useUtcCB">
        <property name="text">
         <string>&amp;Use UTC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dateGB">
     <property name="title">
      <string>Date</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
      <item row="4" column="0">
       <widget class="QLabel" name="dateFormatL">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Date &amp;format</string>
        </property>
        <property name="buddy">
         <cstring>dateFormatCOB</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="dateFormatCOB">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="dontShowDateRB">
        <property name="text">
         <string>&amp;Do not show date</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="showDateBeforeTimeRB">
        <property name="text">
         <string>Show date &amp;before time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="showDateAfterTimeRB">
        <property name="text">
         <string>Show date &amp;after time</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QRadioButton" name="showDateBelowTimeRB">
        <property name="text">
         <string>Show date below time on new &amp;line</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="firstDayOfWeekL">
        <property name="text">
         <string>First day of week in calendar</string>
        </property>
        <property name="buddy">
         <cstring>firstDayOfWeekCB</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="firstDayOfWeekCB">
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="orientationGB">
     <property name="title">
      <string>Orientation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="autorotateCB">
        <property name="text">
         <string>Auto&amp;rotate when the panel is vertical</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>LXQtClockConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>LXQtClockConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dontShowDateRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateFormatL</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dontShowDateRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateFormatCOB</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
